packages/features/bookings/Booker/components/Header.tsx

```diff
@@ -42,7 +43,12 @@ export function Header({
     [setLayout, layout]
   );
 
-  if (isMobile || !enabledLayouts) return null;
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useEffect(() => {
+    setIsEmbed(window?.isEmbed?.() ?? false);
+  });
+
+  if (isMobile || isEmbed || !enabledLayouts) return null;
```

Comment:
Once the files are generated, it will be possible to see the calendar, but `Next.js` complains about a hydration error, and this happens because this component used the window before being defined. Was inserted this `useEffect` because in the previous code the function was called on every render